lib/redis.rb

```diff
@@ -3857,7 +3857,9 @@ def method_missing(command, *args) # rubocop:disable Style/MissingRespondToMissi
     when nil
       {}
     else
+      # rubocop:disable Style/HashTransformValues
```

Comment:
Hum, that's a weird one, I'd have to dig, but I suppose `reply` is an Array here. Might be better to just disable that cop entirely if it has false positives like that.